code/game/objects/items/storage/fancy.dm

```diff
@@ -109,6 +109,7 @@
 	icon_state = "candlebox5"
 	icon_type = "candle"
 	item_state = "candlebox5"
+	w_class = WEIGHT_CLASS_SMALL
 	throwforce = 2
 	slot_flags = ITEM_SLOT_BELT
```

Comment:
```suggestion
```

I recommend simply removing the ability to store in the belt slot instead of changing the other properties of the box. 

This both makes more sense and is more consistent with other boxes that are almost entirely normal sized.

Reply:
I more meant with the fact it makes very little sense to have a box that fits in your belt slot. 

Similarly didn't like First-aid boxes in belts, but it was done for mechanical reasons anyway.

Reply:
I'll axe the belt slot and pop it back to medium/normal-sized when I get home. I'm gonna keep the item restrictions listing though as it still does the "generates more space" thing otherwise (5 small items in a medium box)

Reply:
That is pretty much what I was trying to suggest for it!

Reply:
Hehe brain being bad at interperating things strikes again